addon/-private/system/store.js

```diff
@@ -1783,7 +1783,7 @@ Store = Service.extend({
     @return {boolean}
   */
   recordIsLoaded(modelName, id) {
-    deprecate(`Use of recordIsLoaded is deprecated, use hasRecordForId instead.`, {
+    deprecate(`Use of recordIsLoaded is deprecated, use hasRecordForId instead.`, true, {
```

Comment:
This should be `false` not `true`.  These deprecations (AFAICT) are never being triggered.  

`deprecate('foo', true, { id: 'asdf', until: 'asdf'})` doesn't actually deprecate anything, but `deprecate('foo', false, { id: 'asdf', until: 'asdf'})` does.